### PR TITLE
Document `t_first` as `SUCCESS` response.

### PR DESCRIPTION
**Pull request by stefano-ottolenghi**
@robsdedude and I discovered that `t_first` was undocumented. I took the change to clean up the response metadata as well and alphabetically sort the entries. I copied from the `t_last` entry, but I'm unsure about the version labels @dotStart

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-bolt-65.surge.sh

Updated pages:
- [Bolt Protocol message specification](https://neo4j-docs-bolt-65.surge.sh/bolt/1.0/bolt/message)

File: modules/ROOT/pages/bolt/message.adoc [hunk 1]
```diff
@@ -1256,25 +1256,29 @@ label:new[Introduced in bolt 4.0]
...
+label:deprecated[Removed after bolt 2]
+** `result_available_after::Integer` -- same as `t_first`.
+label:deprecated[Removed after bolt 2]
+** `stats::Dictionary` -- counter information, such as db-hits etc.
 label:new[Introduced in bolt 3]
-** `plan::Dictionary`, plan result.
+** `t_last::Integer` -- the time (in ms) after which the last record in the result stream is consumed.
 label:new[Introduced in bolt 3]
-** `profile::Dictionary`, profile result.
+** `t_first::Integer` -- the time (in ms) after which the first record in the result stream is available.
```
Comment (robsdedude) [inline modules/ROOT/pages/bolt/message.adoc:1278]:
```suggestion
** `t_first::Integer` -- the time (in ms) after which the DBMS was ready to stream the first record in the result.
```

As we've learned it's not the time until the record is available, but until the server has parsed the query, made a plan, etc., and started requesting records from the lower-level DBMS components.

**Comment by robsdedude** [comment]
>  I copied from the `t_last` entry, but I'm unsure about the version labels.

According to the Python drivers code, they've been renamed together going from Bolt 2 to Bolt 3.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-bolt-65.surge.sh

Updated pages:
- [Bolt Protocol message specification](https://neo4j-docs-bolt-65.surge.sh/bolt/1.0/bolt/message)

**Review by robsdedude — APPROVED** [review]
:stopwatch:

**Pull request merged**